src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/config/SnakeCaseFeignConfiguration.java

```diff
@@ -37,7 +42,8 @@ public Encoder feignFormEncoder(
 
     @Bean
     public Decoder feignDecoder() {
-        HttpMessageConverter jacksonConverter = new MappingJackson2HttpMessageConverter(objectMapper);
+        MappingJackson2HttpMessageConverter jacksonConverter = new MappingJackson2HttpMessageConverter(objectMapper);
```

Comment:
Doesn't this bean conflict with the bean created in CamelCaseFeignConfiguration config?